### DISCUSSION TITLE
Seeing a couple UI issues on the latest nightly

### DISCUSSION THREAD

**Discussion by daron81** [orbitalquark/textadept#366] [Q&A]
One issues is with the default Qt GUI version, and another with the curses version.

Issue #1: With the Qt version on Fedora 37, Wayland, after opening a menu from the menu bar and hovering on another root menu, the menu shifts to the top left corner of the screen, no matter where the TA window is positioned.
![Screenshot from 2023-02-23 15-04-27](https://user-images.githubusercontent.com/18105154/221053070-d1e7d78a-0c1e-4929-b110-88cdcc215db3.png)

Issue #2: The color theme of the TA curses version makes comments invisible on the background (for JavaScript, at least).

Any help or ideas would be appreciated. Thanks!

**Comment by orbitalquark** [comment]
I'm pretty sure there's nothing I can do about #1 because that's a Qt 
and/or Wayland issue. Textadept just makes use of Qt and the widgets it 
provides.

As for #2, this one's tough. Textadept inherits the colors your terminal 
defines, and by default, the terminal theme uses bold black for 
comments. In the two terminals I've tried (lxterminal and Konsole), bold 
black is a sort of grey by default, so there is no readability problem 
for me (and presumably anyone else who uses them). You'll probably need 
to either change your terminal colors or change your Textadept terminal 
theme.
